app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsViewModel.kt

```diff
@@ -197,7 +197,7 @@ class TransactionDetailsViewModel
                 var rejectable = false
                 var safeOwner = false
                 if (executionInfo is DetailedExecutionInfo.MultisigExecutionDetails) {
-                    awaitingConfirm = isAwaitingOwnerConfirmation(executionInfo, txDetails!!.txStatus, owners)
+                    awaitingConfirm = false
```

Comment:
Why is it always false and why is isAwaitingOwnerConfirmation() not working correctly here?

Reply:
because we just submitted a confirmation

Reply:
we haven't updated our txDetails and that was the reason isAwaitingOwnerConfirmation was not working correctly